browser/src/control/Control.PartsPreview.js

```diff
@@ -255,6 +255,13 @@ L.Control.PartsPreview = L.Control.extend({
 		}, this);
 
 		L.DomEvent.on(img, 'contextmenu', function(e) {
+			var isMasterView = this._map['stateChangeHandler'].getItemValue('.uno:SlideMasterPage');
+			var $trigger = $('#' + img.id);
+			if (isMasterView && isMasterView === 'true') {
```

Comment:
when it's just a direct variable and not accessing a member you don't need to check if its valid